### PR TITLE
B-01066/contact stubbed out about and began contact

### PR DESCRIPTION
**Pull request by dphamsd**
Check out the controller for contact please!

### CODE REVIEW

File: web/public/js/controller/pages/contact/contact.ctrl.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+auroraApp.controller('ContactCtrl', function($scope, $controller) {
+  angular.extend(this, $controller('LoadableMixin', {$scope: $scope}));
+  	//reserved for possible email and message fields.
```
Comment (jasonmullins) [inline web/public/js/controller/pages/contact/contact.ctrl.js]: This controller doesn't need to extend LoadableMixin - this was just there as a placeholder, but it doesn't need make any asynchronous requests.  You can remove the following timeout block too.

**Comment by jasonmullins** [comment]
You're missing the script tag in main.layout.jade so it can't find the controller.  Ideally, we would use RequireJS or something instead of doing it like this - but we don't have time.

**Comment by dspencer999** [comment]
Looks fine to me.

File: public/js/controller/pages/about/about.ctrl.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 auroraApp.controller('AboutCtrl', function($scope, $controller) {
   angular.extend(this, $controller('LoadableMixin', {$scope: $scope}));
```
Comment (jasonmullins) [inline public/js/controller/pages/about/about.ctrl.js:2]: There's no need to extend LoadableMixin here.  We don't need to get anything asynchronously.  This was just there as an example.

File: public/js/controller/pages/about/about.ctrl.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
...
-
-  // wait for a second to simulate loading
+  	$scope.aboutmessage = 'This web application is designed to present the user with up to date data of food recalls by state.  The system will also alert users via SMS when a new recall has been detected, thereby raising awareness and safety. Additional specs/ technologies used can go here as well.'
+  	$scope.aboutus = 'Some blurb about you guys and your history together.'
+  		// wait for a second to simulate loading
   setTimeout(function () {
```
Comment (jasonmullins) [inline public/js/controller/pages/about/about.ctrl.js:6]: You can remove this whole setTimeout block.  Without extending the loadable mixin, we won't be using the loading pattern.

**Pull request merged**